README.md

```diff
@@ -38,7 +38,7 @@ A common use case could be the following:
 
 `suckit http://books.toscrape.com -j 8 -o /path/to/downloaded/pages/`
 
-[![asciicast](https://asciinema.org/a/327889.svg)](https://asciinema.org/a/327889)
+<a href="https://asciinema.org/a/327889"><img alt="asciicast" src="https://asciinema.org/a/327889.png" width="500" /></a>
```

Comment:
```suggestion
[![asciicast](https://asciinema.org/a/327889.png)](https://asciinema.org/a/327889)
```

Is this not working ?

Reply:
yes but I also wanted to limit image size so I have to scroll less. your call though

Reply:
Ok. Can you center it ?